R/pkg/R/mllib_regression.R

```diff
@@ -110,7 +125,8 @@ setClass("IsotonicRegressionModel", representation(jobj = "jobj"))
 #' @seealso \link{glm}, \link{read.ml}
 setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
           function(data, formula, family = gaussian, tol = 1e-6, maxIter = 25, weightCol = NULL,
-                   regParam = 0.0, var.power = 0.0, link.power = 1.0 - var.power) {
+                   regParam = 0.0, var.power = 0.0, link.power = 1.0 - var.power,
+                   stringIndexerOrderType = "frequencyDesc") {
```

Comment:
I don't think there are corresponding R options for this. One can convert the string into a factor and manipulate the factor easily. It's just the default approach is dropping the first alphabetical category.